Makefile

```diff
@@ -186,18 +191,50 @@ restore-volumes-data:
 
 generate-export-shpfile:
 	-docker-compose exec -T postgresql-geo-api mkdir testdataoutput2
-	-docker-compose exec -T postgresql-geo-api pgsql2shp -f ./testdataoutput2/test.shp -h localhost -p 5432 -r -g the_geom -u ${_GEO_POSTGRES_USER} ${_GEO_POSTGRES_DB} "SELECT the_geom, pug.id as uid, 1 as cost  FROM scenarios_pu_data spd inner join planning_units_geom pug on pug.id = spd.pu_geom_id ";
+	-docker-compose exec -T postgresql-geo-api pgsql2shp -f ./testdataoutput2/test.shp -h localhost -p 5432 -r -g the_geom -u ${GEO_POSTGRES_USER} ${GEO_POSTGRES_DB} "SELECT the_geom, pug.id as uid, 1 as cost  FROM scenarios_pu_data spd inner join planning_units_geom pug on pug.id = spd.pu_geom_id ";
 	-mkdir data/data
 	-docker cp marxan-postgresql-geo-api:testdataoutput2 data/data
 
+
+# Native support tasks
+
+# Create the API and GEO databases. Fails gracefully if the databases already exist.
+native-db-create:
+	@echo "SELECT 'CREATE DATABASE \"${API_POSTGRES_DB}\"' WHERE NOT EXISTS (SELECT FROM pg_database WHERE datname = '${API_POSTGRES_DB}')\gexec" | psql -U "${API_POSTGRES_USER}" -h "${API_POSTGRES_HOST}"
+	@echo "SELECT 'CREATE DATABASE \"${GEO_POSTGRES_DB}\"' WHERE NOT EXISTS (SELECT FROM pg_database WHERE datname = '${GEO_POSTGRES_DB}')\gexec" | psql -U "${GEO_POSTGRES_USER}" -h "${GEO_POSTGRES_HOST}"
+
+# Apply migrations to the current API and GEO databases. Assumes the PostgreSQL server is up.
+native-db-migrate: native-db-create
+	cd api; yarn geoprocessing:typeorm migration:run -t each
+	cd api; yarn api:typeorm migration:run -t each
+
+native-seed-api-init-data:
+	@echo "seeding initial dbs"
+	psql -U "${API_POSTGRES_USER}" -h "${API_POSTGRES_HOST}" ${API_POSTGRES_DB} < api/apps/api/test/fixtures/test-init-apidb.sql
+
+native-seed-geoapi-init-data:
+	@echo "seeding dbs with initial geodata"
+	sed -e "s/\$$user/00000000-0000-0000-0000-000000000000/g" api/apps/api/test/fixtures/test-admin-data.sql | psql -U "${GEO_POSTGRES_USER}" -h "${GEO_POSTGRES_HOST}" ${GEO_POSTGRES_DB}; \
+	sed -e "s/\$$user/00000000-0000-0000-0000-000000000000/g" api/apps/api/test/fixtures/test-wdpa-data.sql | psql -U "${GEO_POSTGRES_USER}" -h "${GEO_POSTGRES_HOST}" ${GEO_POSTGRES_DB};
+	psql -U "${API_POSTGRES_USER}" -h "${API_POSTGRES_HOST}" ${API_POSTGRES_DB} < api/apps/api/test/fixtures/test-features.sql
+	@for i in api/apps/api/test/fixtures/features/*.sql; do \
+		table_name=`basename -s .sql "$$i"`; \
+		featureid=`psql -X -A -t -U "${API_POSTGRES_USER}" -h "${API_POSTGRES_HOST}" ${API_POSTGRES_DB} -c "select id from features where feature_class_name = '$$table_name'"`; \
+		echo "appending data for $${table_name} with id $${featureid}"; \
+		sed -e "s/\$$feature_id/$$featureid/g" api/apps/api/test/fixtures/features/$${table_name}.sql | psql -U "${GEO_POSTGRES_USER}" -h "${GEO_POSTGRES_HOST}" ${GEO_POSTGRES_DB}; \
+		done;
+
+native-seed-api-with-test-data: native-db-migrate native-seed-api-init-data | native-seed-geoapi-init-data
```

Comment:
How those seeds differ from previous? are those related to creating users/projects, while `init-data` pushes wdpa/features and so on?

Reply:
these are copy-paste of the equivalent, non "native-" prefixed tasks above, but just run locally instead of inside a docker container.

Reply:
@tiagojsag sure thing, I just used the opportunity to possibly describe them for future people, not to question its existence/modification

Reply:
Understanding what they do is a fight for another day